echopype/convert/api.py

```diff
@@ -338,8 +338,8 @@ def open_raw(
 
         - ``EK60``: Kongsberg Simrad EK60 echosounder
         - ``ES70``: Kongsberg Simrad ES70 echosounder
-        - ``EK80``: Kongsberg Simrad EK80 and Kongsberg EA640 echsoounders
-        - ``EK80``: Kongsberg Simrad EK80 and Kongsberg EA640 echsoounders
+        - ``EK80``: Kongsberg Simrad EK80 echsoounder
+        - ``EA640``: Kongsberg EA640 echsoounder
```

Comment:
Spelling mistake, echsoounder

Reply:
Hah, thanks for catching this, I guess I wasn't actually reading! :D

Reply:
Done!